packages/cookies/src/response-cookies.ts

```diff
@@ -0,0 +1,110 @@
+import { Cookies } from './cookies'
+import type { CookieSerializeOptions } from 'cookie'
+import * as cookie from 'cookie'
+
+type HasHeaders = { headers: Headers }
+
+const deserializeCookie = (input: HasHeaders): string[] => {
+  const value = input.headers.get('set-cookie')
+  return value !== undefined && value !== null ? value.split(', ') : []
+}
+
+const serializeCookie = (input: string[]) => input.join(', ')
+
+type GetWithOptionsOutput = {
+  value: string | undefined
+  options: { [key: string]: string }
+}
+
+const serializeExpiredCookie = (
+  key: string,
+  options: CookieSerializeOptions = {}
+) =>
+  cookie.serialize(key, '', {
+    expires: new Date(0),
+    path: '/',
+    ...options,
```

Comment:
**issue:** if user calls `delete()` with a custom `expires` in their option, the cookie won't be "deleted"

We should simply put `expires: new Date(0)` after `...options` to guarantee that expiration date will always be past.